app/poros/similar_movie.rb

```diff
@@ -0,0 +1,10 @@
+class SimilarMovie
```

Comment:
Nice work! A potential refactor - could we use the Movie PORO we have to create objects for the similar movies also?